public-ui/src/app/data-browser/views/ehr-view/ehr-view-react.component.tsx

```diff
@@ -0,0 +1,636 @@
+                       </div>
+                       {(concepts.length === 1 && concepts[0].standardConcept !== 'S' && standardConcepts.length > 0) &&
+                       <div className='db-alert' style={styles.dbAlert}>
+                        Note: {concepts[0].vocabularyId} {concepts[0].conceptCode} '{concepts[0].conceptName}'
+                            maps to Standard Vocabulary {standardConcepts[0].vocabularyId}
+                            {standardConcepts[0].conceptCode} '{standardConcepts[0].conceptName}'.
+                            Standard vocabularies capture data across a variety of source vocabularies.
+                       </div>
+                       }
+                       <div className='tbl-r tbl-head'>
+                        <div className='tbl-d body-lead'> {domain.name}
+                            <TooltipReactComponent tooltipKey={domain.domain.toLowerCase()}
+                                                             label='EHR Tooltip Hover' searchTerm={searchWord}
+                                                             action='Domain name tooltip hover in matching concepts table' />
+                        </div>
+                        <div className='tbl-d body-lead'> Participants of {totalParticipants.toLocaleString()}
+                                      <TooltipReactComponent tooltipKey='participantCountHelpText' label='EHR Tooltip Hover'
+                                                         searchTerm={searchWord} action='Participant count tooltip hover' />
+                        </div>
+                        <div className='tbl-d body-lead'> % of {totalParticipants.toLocaleString()}
+                          <TooltipReactComponent tooltipKey='percentageOfParticipants'
+                                             label='EHR Tooltip Hover' searchTerm={searchWord}
+                                             action='Percentage of participant count tooltip hover' />
+                        </div>
+                        {(domain.domain.toLowerCase() === 'measurement') &&
+                        <div className='tbl-d body-lead info-text'>
+                          Data Type
+                          <div className={dropdownClass}>
+                            <button className='dropdown-toggle'>
+                              <ClrIcon shape='filter-grid' className={filterIconClass} onClick={() =>
+                              this.setState({selectedMeasurementTypeFilter: !selectedMeasurementTypeFilter})} />
+                            </button>
+                            <div className='dropdown-menu'>
+                                <div className='clr-checkbox-wrapper'>
+                                    <div className='checkbox-input'>
+                                        <input type='checkbox' id='checkbox1' className='clr-checkbox' onClick={() =>
+                                        this.setState({measurementTestFilter: !measurementTestFilter},
```

Comment:
@dolbeew This is the place where the measurementtestfillter/ ordefilter value changes.